test/data/installer/capsule-installer/config_header.txt

```diff
@@ -0,0 +1,8 @@
+# Format:
```

Comment:
Nitpick: having `test/definitions/data` would be a better place for definitions-specific mock data IMO, but leaving the decision up to you

Reply:
@iNecas I had it there originally. Then I added some tests for the hammer utils and while it called the feature internaly I had to mock the configs there so I moved the data to common space. 

Is it allowed practice to use definitions from the `lib`? For Hammer utils it made sense as a short term solution but could be avoided. Testing the other parts e.g. `cli` might also benefit from common data for mock ups. Saying that these data aren't that definition-specific so it makes sense to keep it shared among lib and def's tests.

Reply:
:+1: then